### DISCUSSION TITLE
Multiselect Without Chip GET Request

### DISCUSSION THREAD

**Discussion by iamjohnford** [josefarias/hotwire_combobox#154] [Ideas]
Is there a way to load a multiselect combobox without needing to make a request to the server for the chip? Like having a `multiselect_chip_template` that renders the HTML template inline and is used for the chip in the JavaScript? I searched through the code but wasn't able to find anything like that.

One reason this came up was I ran into some challenges with my slightly unique scenario. The values in my list can be long. When selecting a bunch of them, the GET request passes those as part of the query string to grab the chips, and eventually maxes out the query string limit.

One option that could help my scenario would be using POST for the chips instead of GET. Another would be having everything inline and not need to make requests for chips.

The tool has been super helpful and I really appreciate you sharing all of your hard work!

**Comment by josefarias** [comment]
Thanks for bringing this up!

Yeah I can see this also being a problem for folks using UUIDs as identifiers.

I don't think I'd like us to have more than one way to render the chips. So not really favoring the inline solution right now.

However, I think we can make POST work semantically by switching some things around. I'll try to get to this soon. I'm confident we'll find a solution.

**Comment by josefarias** [comment]
POST route implemented here: https://github.com/josefarias/hotwire_combobox/pull/162

Will be available in the next release. Closing for now, but please let me know if that didn't fix your long values problem.

Thanks! Great idea using POST.
